src/common/codeUtil.ts

```diff
@@ -48,4 +51,85 @@ export class CodeUtil {
 
         return regs;
     }
+
+    /** Align variable assignment by = operator in selected text
+     * @param selection Text selection in editor
+     */
+    public static alignTextAssignOperator(selection: Selection): string {
+        const document = vscode.window.activeTextEditor.document;
+        let maxPosition = 0; // Right-most = operator position in line from all assignments
+        for (
+            let lineIndex = selection.start.line;
+            lineIndex <= selection.end.line;
+            lineIndex++
+        ) {
+            const line = this.normalizeLineAssignOperator(
+                document.lineAt(lineIndex).text,
+            );
+
+            // Find right-most = operator position
+            let position = line.search('='); // = operator position
```

Comment:
If possible, `indexOf` should be a bit more performant (this is just a guess 🙃 but it's definitely more familiar than `string.search` to most folks)

```suggestion
            let position = line.indexOf('='); // = operator position
```